docs/resources/uberbank_example1.ini

```diff
@@ -21,8 +21,14 @@ file-retention-level = merged_triggers
 
 ; Start and end times are needed to obey LIGO file naming conventions. Actual
 ; values are irrelevant. This can be overriden on the command line.
-start-time = 900000000
-end-time = 900010000
+; Here we use the times matching the estimated PSD so we have an immediate
+; reference on the resulting bank file name
+start-time = 1126051217
+end-time = 1137254417
+
```

Comment:
Sorry, I tend to put a comment *above* the line I'm commenting on (looks more natural in the "files changed" window). This is referring to lines 29 - 31 (max-bh-spin to nsbh-boundary)